action.yml

```diff
@@ -47,13 +47,11 @@ runs:
     - name: Fetch metadata
       id: dependabot-metadata
       uses: dependabot/fetch-metadata@v1
-      if: github.event_name == 'pull_request' && (github.actor == 'dependabot[bot]' || inputs.skip-verification == 'true')
```

Comment:
I'm not sure about the removal of this, since we don't control this action there's a chance that the action fails if executed without this condition. to be sage, I would just remove the condition from the execution of our own action, but also make sure that the same logic is applied within the action js code.

as for why this may have been added, hard to say, but I guess to short-circuit and avoid execution if unnecessary.

ideally I'd like to remove the yaml condition in both actions, but I fear that this action we don't control will break

Reply:
yep, that makes sense.